Doc 1281/changelog 18 sept (#420)
swan-io/docs: changelog/coming-up.mdx

📖 Starting **October 9th**, the new **[Verification of Payee](/preview/vop/verification-of-payee/)** regulation takes effect.

📡 Starting **October 9th**, bulk SEPA Credit Transfers from individual account holders will be rejected if [verification tokens](/preview/vop/guide-verify-beneficiary/#step-3) aren't included in the payment initiation. You must either implement the `verifyBeneficiary` mutation (recommended) or restrict bulk credit transfer access to company accounts only. This change applies to both Live and Sandbox environments.

💳 On **October 16th**, the `CardUrl` field will be removed. This field is being deprecated. Use `cardMaskedNumber`, `expiryDate`, and `cardDesignUrl` instead.

🛒 On **October 16th**, the following [merchant profiles and merchant payment methods](/topics/merchants/) breaking changes will take effect:

> - A new `WaitingForInformation` status and `verificationRequirements` list will be available for merchant profiles and Merchant Payment Methods to communicate missing information. These requirements must be provided at the merchant profile level using the `requestMerchantProfile` and `requestMerchantProfileUpdate` mutations. 

Reviewer: ```suggestion
- A new `WaitingForInformation` status and `verificationRequirements` list will be available for merchant profiles and merchant payment methods to communicate missing information. These requirements must be provided at the merchant profile level using the `requestMerchantProfile` and `requestMerchantProfileUpdate` mutations. 
```

Author: Yes. Done.

> - A new `supportingDocumentCollections` field will also be available at the merchant profile level for providing required documents.

Reviewer: Yes similar to this - this is clearer what exactly the impact is.

- The fields `invoiceExample`, `expectedMonthlyPaymentVolume`, `merchantWebsite`, `socialNetwork`, `supportWebsite`, `termsAndConditions` will be **removed** from the API.

> - `rejectReasons` will be removed for Card Payment Methods.

Reviewer: ```suggestion
- `rejectReasons` will be removed for card payment methods.
```

> - `rejectReasons` will be removed for Card Payment Methods.

Reviewer: Following other breaking changes - should we add the "will be removed. Use XYZ from now on"?

Reviewer: So we're changing how we usually talk about upcoming breaking changes? By splitting them out? The XYZ's have been defined they're just mentioned in the Improvements section.

> - `rejectionReasons` for `Rejected` merchant profiles and Merchant Payment Methods will become mandatory in the Testing APIs `simulateMerchantProfileRequestOutcome` and `simulateMerchantPaymentMethodRequestOutcome`.

Reviewer: ```suggestion
- `rejectionReasons` for `Rejected` merchant profiles and merchant payment methods will become mandatory in the Testing APIs `simulateMerchantProfileRequestOutcome` and `simulateMerchantPaymentMethodRequestOutcome`.
```

- Dynamic rolling reserve changes and maximum payment amount limits will be introduced at the merchant payment method level to mitigate risk. Subscribe to the `MerchantPaymentMethod.Created` and `MerchantPaymentMethod.Updated` webhooks for these updates.

💸 Effective **October 19th**, we're introducing the following updates to the `initiateCreditTransfers` endpoint to improve its usability and clarity:

- The `save` parameter will default to `false`.
- The `mode` parameter will be a **required** field. The default value is set to `Regular`.
- The `isMyOwnIban` and `beneficiaryId` input fields will be **removed**.
- The API will return a `ValidationRejection` error instead of `ForbiddenRejection` when an input contains **invalid** data.

> ⚠️ Starting **October 31, 2025**, the `taxIdentificationNumber` will be a mandatory field at onboarding when an account holder's residency country does not match the `accountCountry`. This is to comply with [CRS (Common Reporting Standard) regulations](https://www.oecd.org/content/dam/oecd/en/topics/policy-issue-focus/aeoi/jersey-guidance-notes-crs.pdf).

Reviewer: ```suggestion
⚠️ Starting **October 31, 2025**, the `taxIdentificationNumber` will be a mandatory field at onboarding when an account holder's residency country doesn't match the `accountCountry`. This is to comply with [CRS (Common Reporting Standard) regulations](https://www.oecd.org/content/dam/oecd/en/topics/policy-issue-focus/aeoi/jersey-guidance-notes-crs.pdf).
```
